tiup/tiup-command-uninstall.md

```diff
@@ -0,0 +1,36 @@
+---
+title: tiup uninstall
+---
+
+# tiup uninstall
+
+The `tiup uninstall` command is used to uninstall the installed components.
+
+## Syntax
+
+```sh
+tiup uninstall <component1>:<version> [component2...N] [flags]
+```
+
+- `<component1>`: the name of a component to uninstall.
```

Comment:
When we use this command, we are very clear which component we are gonna uninstall.
```suggestion
- `<component1>`: the name of the component to uninstall.
```